.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Ensure labels"
+on: # yamllint disable-line rule:truthy
+    pull_request:
+        types:
+            - labeled
+            - unlabeled
+            - opened
+            - synchronize
+            - reopened
+            - ready_for_review
+
+jobs:
+    changelog:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check changelog label
+              if: github.event.pull_request.draft == false && false == contains(join(github.event.pull_request.labels.*.name, ','), 'changelog/')
+              run: |-
+                  echo "::error::Add 'changelog/*' label";
+                  exit 1;
+            - name: Check resource label
+              if: github.event.pull_request.draft == false && false == contains(join(github.event.pull_request.labels.*.name, ','), 'changelog/no-changelog') && false == contains(join(github.event.pull_request.labels.*.name, ','), 'resource/')
```

Comment:
we can skip this check also for `changelog/note`

Reply:
alternatively we could use PR title prefix `[{resource_name}] ...`

Reply:
Yeah... It's a bit harder to verify it's in a list fo known names.